ironfish/src/rpc/routes/wallet/getBalances.test.ts

```diff
@@ -85,18 +85,137 @@ describe('Route wallet/getBalances', () => {
       })
 
       expect(getBalances).toHaveBeenCalledTimes(1)
-      expect(response.content.balances).toEqual(
-        mockBalances.map((mockBalance) => ({
-          ...mockBalance,
-          assetId: mockBalance.assetId.toString('hex'),
-          assetName: mockBalance.assetName.toString('hex'),
-          assetOwner: mockBalance.assetOwner.toString('hex'),
-          confirmed: mockBalance.confirmed.toString(),
-          unconfirmed: mockBalance.unconfirmed.toString(),
-          pending: mockBalance.pending.toString(),
-          available: mockBalance.available.toString(),
-        })),
+
+      expect(response.content.balances).toEqual([
+        {
+          ...mockBalances[0],
+          assetId: mockBalances[0].assetId.toString('hex'),
+          assetName: mockBalances[0].assetName.toString('hex'),
+          assetOwner: mockBalances[0].assetOwner.toString('hex'),
+          assetVerification: { status: 'unknown' },
+          confirmed: mockBalances[0].confirmed.toString(),
+          unconfirmed: mockBalances[0].unconfirmed.toString(),
+          pending: mockBalances[0].pending.toString(),
+          available: mockBalances[0].available.toString(),
+        },
+        {
+          ...mockBalances[1],
+          assetId: mockBalances[1].assetId.toString('hex'),
+          assetName: mockBalances[1].assetName.toString('hex'),
+          assetOwner: mockBalances[1].assetOwner.toString('hex'),
+          assetVerification: { status: 'unknown' },
+          confirmed: mockBalances[1].confirmed.toString(),
+          unconfirmed: mockBalances[1].unconfirmed.toString(),
+          pending: mockBalances[1].pending.toString(),
+          available: mockBalances[1].available.toString(),
+        },
+      ])
+    })
```

Comment:
Can we revert this test? I'm not sure it needs to change. We can also use toMatchObject() which is better in jest because it means other tests don't need to change in the case that you want to test a new field. Otherwise, every new field has to change all tests that use toEqual() which is why it's not recommended to be used.